packages/qwik/src/cli/migrate-v2/run-migration.ts

```diff
@@ -0,0 +1,35 @@
+import { confirm, intro, isCancel, log } from '@clack/prompts';
+import type { AppCommand } from '../utils/app-command';
+import { bgMagenta, green } from 'kleur/colors';
+import { bye } from '../utils/utils';
+import { replacePackage } from './replace-package';
+import { updateDependencies } from './update-dependencies';
+import { versions } from './versions';
+
+export async function runV2Migration(app: AppCommand) {
+  intro(
+    `✨  ${bgMagenta(' This command will migrate your Qwik application from v1 to v2 \n')}` +
+      `This includes the following: \n` +
+      //   TODO(migrate-v2): package names
+      `  - "@builder.io/qwik", "@builder.io/qwik-city" packages will be rescoped to "@qwik.dev/core" and "@qwik.dev/city" \n` +
+      `  - related dependencies will be updated \n`
+  );
+  const proceed = await confirm({
+    message: 'Do you want to proceed?',
+    initialValue: true,
+  });
+
+  if (isCancel(proceed) || !proceed) {
+    bye();
+  }
+
+  try {
+    replacePackage('@builder.io/qwik-city', '@qwik.dev/city', versions['@qwik.dev/city']);
+    replacePackage('@builder.io/qwik', '@qwik.dev/qwik', versions['@qwik.dev/qwik']);
```

Comment:
should we add `@builder.io/qwik-auth` -> `@auth/core` as well? considering it was removed in v2

Reply:
yes please add

Reply:
@shairez anything related to `qwik-labs` or other packages to add here?

Reply:
so... I think `qwik-auth` should be migrated to `@auth/qwik` but there are a bunch of extra migration steps for that, so I don't know if we want to get into that via this script.

Regarding `qwik-labs`, all of its features have been folded into either `core` or `router`.
But the only used one I know of is `insights`.

I've documented its changes here - 
https://github.com/QwikDev/qwik/pull/7018

again, it's not widely used, so I don't know if it worth the effort to make it automatic like the other more used ones.